lib/index.d.ts

```diff
@@ -669,6 +660,30 @@ declare namespace Stardog {
              */
             function get(conn: Connection, database: string, fileName: string, params?: object): Promise<HTTP.Body>;
         }
+
+        namespace namespaces {
+          /**
+           * Gets a mapping of the namespaces used in a database.
+           *
+           * @param {Connection} conn the Stardog server connection
+           * @param {string} database the name of the database
+           * @param {object} params additional parameters if needed
+           */
+          function get(conn: Connection, database: string, params?: object): Promise<HTTP.Body>;
+
+          /**
+           * Extracts namespaces from an RDF file or RDF string and adds any
+           * new namespaces to those that already exist in the database.
```

Comment:
should you clarify that existing prefixes are updated if the namespace changed?

Reply:
and I forget - is there a doc file that gets generated with this?

Reply:
https://github.com/stardog-union/stardog.js/blob/master/package.json#L115

Reply:
Sure, changed.